requirements/secure.txt

```diff
@@ -0,0 +1,6 @@
+-r ./dev.txt
```

Comment:
As we discusses, there no need to recursively include dev.txt since `secure` is specific and rare use case.